Mathlib/Analysis/Calculus/Implicit.lean

```diff
@@ -177,11 +179,26 @@ theorem implicitFunction_apply_image :
     ∀ᶠ x in 𝓝 φ.pt, φ.implicitFunction (φ.leftFun x) (φ.rightFun x) = x :=
   φ.hasStrictFDerivAt.eventually_left_inverse
 
+theorem leftFun_implicitFun_mixed_args : ∀ᶠ x in 𝓝 φ.pt,
```

Comment:
`mixed_args` has no obvious meaning in the naming convention. I'd just go for `leftFun_implicitFun` (what else could you apply it to?) 

Also, since the name of the object is `implicitFunction`, it's always better for ease of discoverability to use the exact same name in the lemma name. In the end, I would go for `leftFun_implicitFunction`.

Same thing for the next lemma.